### DISCUSSION TITLE
Helper functions for data management keys

### DISCUSSION THREAD

**Discussion by jmcdo29** [pactumjs/pactum#135] [Ideas]
It would be nice if we had a helper function to generate the keys for the data management. Ran into a small bug earlier because I forgot a `{` when working with a template literal (something along the lines of 
```ts
`$S${someVariable}}`
```
when it should have had one more `{` before the second `$`. To alleviate this now I've made a helper function to generate the keys for me
```ts
export const getStashKey = (key: string) => `$S{${key}}`
```
And can do the same for the data maps and function.

While we're at it, could we also export some constants for `@OVERRIDES@` and `@DATA:TEMPLATE@`? I've made some in my local code, but figure it could be nice to have it provided by the library as well

**Comment by ASaiAnudeep** [comment]
Yes, makes sense. 

Original idea was to create a plugin in vs-code for highlighting the special syntax from pactum. For a quick fix, as you suggested we can include some helper methods for avoiding typos in data management.
